diagnostic.md

```diff
@@ -73,14 +132,35 @@ data in JSON format.
 ```
 
 ```sh
-# your answer here
+curl --include --request POST "http://localhost:4741/donuts" \
+  --header "Content-Type: application/json" \
+  --data '{
+    "donuts": {
+      "name: "French Cruller",
+      "price": "$0.99"
+    }
+  }'
 ```
 
 Write an AJAX request to create a single donut on the server using JSON. Please
 do not use `data = getFormFields(form)` instead write out the data object.
 
 ```js
-let createDonut = /* your answer here */;
+
+const updateDonut= function(event){
+  event.preventDefault()
+  const data = ..
+  const donut = data.donut
+}
+
+
+let create = fucntion(data){
+  return $.ajax({
```

Comment:
Should look like this: 
`curl --include --request "POST" "http://example.com/donuts" \
  --header "Content-Type: application/json" \
  --data $'{
    "donut": {
      "name": "French Cruller",
      "price": "$0.99"
    }
  }'`